### ISSUE TITLE
example for createSourceEventStream from 'graphql/subscription'

### ISSUE THREAD

**Issue by Elaman19** [graphql/graphql-js#1004]
Hi! Can you write example how to use createSourceEventStream from 'graphql/subscription'?
Thanks.

**Comment by robzhu** [comment]
Is a createSourceEventStream example all you need? Would a more comprehensive end-to-end example be more useful?

**Comment by Elaman19** [comment]
Yes, i need more comprehensive end-to-end example. Can you write or give a link?

**Comment by adjourn** [comment]
I would also love to see this, especially a minimal code/parts to "make it work".

I really appriciate packages like [graphql-subscriptions](https://github.com/apollographql/graphql-subscriptions) or [subscriptions-transport-ws](https://github.com/apollographql/subscriptions-transport-ws) but there's not much explained, a lot of noise & they define schema differently (e.g `resolvers` object) which makes it even more difficult to understand.

E.g if you use these packages:
`redis pub/sub -> graphql(graphql-subscriptions) -> subscriptions-transport-ws -> *internet* -> client`
It's not very clear why do I even need `graphql-subscriptions` or what it exactly does.

**Comment by chaitanya11** [comment]
Does anyone has anything on this issue ?
